build-system/global-configs/canary-config.json

```diff
@@ -8,7 +8,6 @@
   "expDoubleclickA4A": 1,
   "pan-y": 1,
   "a4aProfilingRate": 1,
-  "amp-form-custom-validations": 1,
```

Comment:
yes, will need to wait until the guard removal hits prod to not unlaunch the feature